mass_mailing_list_dynamic/models/mail_mass_mailing_list.py

```diff
@@ -55,7 +55,7 @@ def action_sync(self):
             current_partners = current_contacts.mapped("partner_id")
             # Add new contacts
             for partner in desired_partners - current_partners:
-                Contact.create({
+                Contact.with_context(auto_created=True).create({
```

Comment:
Can you provide here a comment pointing to why it's done this way. I have read your main comment and I still don't understand it...

Reply:
in an other module i extend the create function of mail.mass_mailing.contact to automatically send a double opt in mail to the new created contact. To be able to not send it, in the case the contact got created in some other way - i've added this context var.

Reply:
Which module?